ikarus/solver/nonlinearsolver/newtonraphsonwithscalarsubsidiaryfunction.hh

```diff
@@ -143,18 +138,19 @@ namespace Ikarus {
         nonLinearOperator().updateAll();
         rNorm = sqrt(rx.dot(rx) + subsidiaryArgs.f * subsidiaryArgs.f);
 
-        this->notify(NonLinearSolverMessages::SOLUTION_CHANGED);
+        this->notify(NonLinearSolverMessages::SOLUTION_CHANGED, lambda);
```

Comment:
maybe we could also pass here simply a more general struct with all information similar to `solverInformation`. Basically something, where the current state can be accessed. Can you write an issue since this does not need to be done now